nautobot/dcim/templates/dcim/cable.html

```diff
@@ -85,6 +85,7 @@ <h1>{% block title %}Cable {{ object }}{% endblock %}</h1>
             </div>
             {% include 'inc/custom_fields_panel.html' %}
             {% include 'inc/relationships_panel.html' %}
+            {% include 'inc/computed_fields_panel.html' %}
```

Comment:
Nit: it feels to me like it would make sense for computed fields to be adjacent to custom fields, not separated by the relationships panel.